src/Robo/Commands/Acsf/AcsfCommand.php

```diff
@@ -71,6 +71,12 @@ public function acsfInitialize($options = ['acsf-version' => '^2.47.0']) {
       $project_config['modules']['local']['uninstall'][] = 'acsf';
     }
     YamlMunge::writeFile($project_yml, $project_config);
+
+    // Exclude .htaccess from further updates due to ACSF patch.
```

Comment:
Is there a reference to more text? I can't tell what ACSF patch this is referring to.

Reply:
Oh, now from the comments I get it. It's about ACSF patch of .htaccess. Can we just say in the comment `// .htaccess was patched, excluding from further updates` if that is the case?